src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,115 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.TheSystemsNodes.ClimbingNode;
+import edu.greenblitz.tobyDetermined.Nodesssss.TheSystemsNodes.GriperNode;
+import edu.greenblitz.tobyDetermined.Nodesssss.TheSystemsNodes.NodeArm;
+import edu.wpi.first.math.Pair;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+
+import static edu.greenblitz.tobyDetermined.Nodesssss.NodeBase.SystemsPos.*;
+
+public class NodeBase {
+    public enum SystemsPos {
+        GRIPER_CLOSE,
+        GRIPER_OPEN,
+
+        CLIMBING_ROMY,
+        CLIMBING_NOAM,
+
+        ARM_LOW,
+        ARM_MID,
+        ARM_HIGH,
+        ARM_ZG,
+
+        MID_NODE;
+
+    }
+
+    public static class CreateNodes {
```

Comment:
I know there was some kind of Git messup so I won't be too strict on this but the initialization done here is out of scope for a PR called "**Generic** Nodes"